### PR TITLE
Cookie validation > #46 #48

### PR DESCRIPTION
**Pull request by BenNewman90**
- [ ] makes a fetch request to the server, which validates the cookie and responds with the decrypted cookie. Currently stored in the state. This should change, but have moved onto other things as it was become a major block.
- [ ] did a bit more on the 404 page
- [ ] Refactored App.js and moved the file outside of the components

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -18,11 +18,6 @@ app.use(cookieParser());
...
 app.use(bodyParser.json());
 
-app.use((req, res, next) => {
-  console.log(req.cookies);
-  next();
-});
-
 app.get('/api/users', (req, res) => {
   getUsers()
     .then(users => res.json(users))
```
Comment (ghassanmas) [inline server/server.js]: Is the data in getUser() should be displalyed only for logged in users?, if so, should check the cookies before responding to the client with data

Comment (BenNewman90) [reply server/server.js]: No, it's not sensitive. It's used to display the table.

Comment (ghassanmas) [reply server/server.js]: isn't the table is in the prediction page which only logged in user should see ?

Comment (ghassanmas) [reply server/server.js]: Aha I see so you don't have any requests that needs the user to be logged in yet beside `api/cookieValidation`

**Pull request merged**